static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -645,6 +724,15 @@ const CheckboxCellContainer = styled('div')`
   margin: 0 -${space(1)} 0 -${space(0.5)};
 `;
 
+const CheckboxClickTarget = styled('label')`
+  cursor: pointer;
+  display: block;
+  margin: -${p => p.theme.space.md};
+  padding: ${p => p.theme.space.md};
+  max-width: unset;
+  line-height: 0;
+`;
```

Comment:
This is here to give the checkbox a little extra space to capture clicks. 

The checkbox is a small target, and missing it means a whole pageload :(